lib/games/find_size_game.dart

```diff
@@ -0,0 +1,351 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:maui/components/responsive_grid_view.dart';
+import 'package:maui/components/shaker.dart';
+import 'package:maui/components/unit_button.dart';
+import 'package:maui/games/single_game.dart';
+import 'package:maui/state/button_state_container.dart';
+
+class FindSizeGame extends StatefulWidget {
+  Function onScore;
+  Function onProgress;
+  Function onEnd;
+  int iteration;
+  GameConfig gameConfig;
+  bool isRotated;
+
+  FindSizeGame(
+      {key,
+      this.onScore,
+      this.onProgress,
+      this.onEnd,
+      this.iteration,
+      this.gameConfig,
+      this.isRotated = false})
+      : super(key: key);
+  @override
+  _FindSizeGameState createState() => _FindSizeGameState();
+}
+
+class _FindSizeGameState extends State<FindSizeGame>
+    with TickerProviderStateMixin {
+  String dropBoxBg = "assets/box.jpg";
+  List<String> _dropData = new List();
+  List<String> _images = [
+    "assets/apple.png",
+    "assets/Colors.png",
+    "assets/Boy.png"
+  ];
+
+  int code;
+  bool _isLoading = true;
+
+  List<bool> _isDone = new List();
+  var data, flag1, correct, keys;
+
+  initState() {
+    super.initState();
+    initfn();
+  }
+
+  initfn() {
+    setState(() => _isLoading = true);
+    // data = await fetchWordData(
+    //   widget.gameConfig.gameCategoryId, _maxSize, _otherSize);
+    int temp = 0;
+    while (temp < _images.length) {
+      if (temp < _images.length) _isDone.add(false);
```

Comment:
Why is this needed? If you have a while loop with a condition the same condition inside will always be true